nixos/modules/services/monitoring/prometheus/exporters/smartctl.nix

```diff
@@ -41,13 +42,23 @@ in {
   serviceOpts = {
     serviceConfig = {
       AmbientCapabilities = [
+        "CAP_RAW_SYSIO"
         "CAP_SYS_ADMIN"
       ];
       CapabilityBoundingSet = [
+        "CAP_RAW_SYSIO"
```

Comment:
Shouldn't this be `CAP_SYS_RAWIO`?

Reply:
Did you review this PR before merging it? :smile: